CRM/Contact/BAO/Query.php

```diff
@@ -3899,7 +3899,21 @@ public function changeLog(&$values) {
     $name = $targetName[4] ? "%$name%" : $name;
     $this->_where[$grouping][] = "contact_b_log.sort_name LIKE '%$name%'";
     $this->_tables['civicrm_log'] = $this->_whereTables['civicrm_log'] = 1;
-    $this->_qill[$grouping][] = ts('Modified By') . " $name";
+    $fieldTitle = ts('Added By');
+    foreach ($this->_params as $params) {
+      if ($params[0] == 'log_date') {
+        if ($params[2] == 2) {
+          $fieldTitle = ts('Modified By');
+        }
+        break;
+      }
+    }
+    list($qillop, $qillVal) = self::buildQillForFieldValue(NULL, 'changed_by', $name, 'LIKE');
+    $this->_qill[$grouping][] = ts("%1 %2 %3", [
```

Comment:
@pradpnayak can you  add single quotes for value:
```php
$this->_qill[$grouping][] = ts("%1 %2 '%3'", [
...
```

Reply:
Thanks @monishdeb 

Pushed the change.

Reply:
@monishdeb looks like your review has been responded to - good to merge now?